src/wms/src/main/java/org/geoserver/wms/capabilities/GetCapabilitiesTransformer.java

```diff
@@ -692,9 +692,13 @@ private void handleLayers() {
             }
 
             //WMSInfo serviceInfo = wmsConfig.getServiceInfo();
-            element("Title", serviceInfo.getTitle());
-            element("Abstract", serviceInfo.getAbstract());
-
+            if(StringUtils.isBlank(serviceInfo.getRootLayerAbstract())) {
```

Comment:
As above, since the two fields are independant, the checks for title and abstract should probably be seperate